bc/web/templates/homepage.html

```diff
@@ -20,7 +20,7 @@ <h1 class="md:max-w-md xl:max-w-xl font-[800] text-5xl md:text-6xl lg:text-6xl x
         <div class="py-8 md:py-16 text-white">
             <div class="grid grid-cols-2 flex flex-col justify-center gap-10 pt-6">
                 <div class="flex justify-start self-start">
-                    <img src="{% static "svg/chat-app-logos-combined.svg" %}" class="w-48 object-contain sm:w-64 md:w-72 lg:w-80 xl:w-[490px] xl:h-[440px]">
+                    <img alt='Chat app logos combined' src="{% static "svg/chat-app-logos-combined.svg" %}" class="w-48 object-contain sm:w-64 md:w-72 lg:w-80 xl:w-[490px] xl:h-[440px]">
```

Comment:
We can do better here. How about "logos for popular chat applications, including Slack, Discord, Microsoft Teams and Google Chat"

Reply:
You're right! That's a lot better